crates/turborepo-lib/src/commands/logout.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
+use log::error;
+
+use crate::{
+    config::{default_user_config_path, UserConfig},
+    ui::{GREY, UI},
+};
+
+pub fn logout(ui: UI) -> Result<()> {
+    let mut config = UserConfig::load(
+        &default_user_config_path()
+            .ok_or_else(|| anyhow::anyhow!("Unable to find default config path"))?,
```

Comment:
```suggestion
        &default_user_config_path()
            .context("Unable to find default config path")?,
```

something anyhow provides